config/locales/de.yml

```diff
@@ -927,7 +927,7 @@ de:
           show: Anzeigen
           upload_pending: Dateien werden gerade hochgeladen.
         other_entries_blank_slate:
-          none_available: Keine anderen Beiträge verfügbar
+          none_available: Keine anderen Beiträge verfügbar.
```

Comment:
Locale changes have to go through LocaleApp. See also  [localization wiki page](https://github.com/codevise/pageflow/wiki/Editing-Translations)